test/e2e/delegation.js

```diff
@@ -62,11 +62,13 @@ test(`delegation`, async function(t) {
   t.test(`Stake`, async t => {
     let totalAtoms = (await app.client
       .$(`.header-balance .total-atoms h2`)
-      .getText()).split(`.`)[0] // 130.000...
+      .getText()).split(`.`)[0] // 30.000...
```

Comment:
huh? why is it suddenly 100 less?

Reply:
I think bonded atoms are not calculated in to the total atoms any longer. Needs to be fixed.

Reply:
Exaclty this point I was a uncertain on how this changed!

Reply:
Still an issue?

Reply:
fixed! 130.000 again

Reply:
by replacing `state.stuff = x` with `Vue.set(state, 'stuff', x)`